Comments/ClientApp/src/components/Review/Review.js

```diff
@@ -326,6 +326,7 @@ export class Review extends Component<PropsType, StateType> {
 				console.log(err);
 				if (err.response) alert(err.response.statusText);
 			});
+		window.scrollTo(0,0);
```

Comment:
we don't need this any more because the 'submitted' page is in a separate route now